### PR TITLE
remove extra console logs

### PR DESCRIPTION
**Pull request by divine-comedian**
remove extra console logs to optimize performance https://github.com/Giveth/devops/issues/195

### CODE REVIEW

File: src/components/views/donate/DonateModal.tsx [hunk 1]
```diff
@@ -174,12 +173,10 @@ const DonateModal: FC<IDonateModalProps> = props => {
 									delayedCloseModal(firstHash, secondHash);
 								return;
 							}
-							console.log('SecondDonation Success', secondHash);
 							setSecondTxStatus(EToastType.Success);
 							isSaved && delayedCloseModal(firstHash, secondHash);
 						})
 						.catch(({ txHash: secondHash }) => {
-							console.log('SecondDonation Error', secondHash);
```
Comment (RamRamez) [inline src/components/views/donate/DonateModal.tsx:182]: @divine-comedian We still have some bugs on donation page, I suggest we keep logs related to DonateModal component

File: src/lib/helpers.ts [hunk 2]
```diff
@@ -335,7 +332,6 @@ async function handleEthTransfer(params: TransactionParams): Promise<Address> {
 		data: '0x',
 	});
 
-	console.log('ETH transfer result', { hash });
```
Comment (RamRamez) [inline src/lib/helpers.ts:338]: likewise

**Review by alireza-sharifpour — APPROVED** [review]
Thanks @divine-comedian

**Review by MohammadPCh — APPROVED** [review]
@divine-comedian Thanks, LGTM

**Pull request merged**